src/App/mermaidData/syncApiDataIntoOfflineStorage/SyncApiDataInfoOfflineStorage.test.js

```diff
@@ -800,3 +817,35 @@ test('pushChanges includes the expected modified data', async () => {
 
   expect(response).not.toBeUndefined()
 })
+
+test('All of the push functions handle sync errors with the callback function', async () => {
```

Comment:
what is the callback function, can we be more specific?

Reply:
great call